sportsdataverse/mlb/retrosheet.py

```
@@ -24,16 +24,17 @@ def retrosheet_ballparks() -> pd.DataFrame():
         None
     
     Returns:
-        A pandas Dataframe with the biographical information of notable major leauge teams.
+        A pandas Dataframe with the biographical information of notable major league teams.
     """
-    people_url = "https://www.retrosheet.org/parkcode.txt"
-    resp = download(people_url)
-    resp_str = str(resp, 'UTF-8')
-    resp_str = StringIO(str(resp, 'UTF-8'))
-    park_df = pd.read_csv(resp_str,sep=",",)
-    park_df.columns=['park_id','park_name','park_alt_name','park_city',
-    'park_state','park_start_date','park_end_date','park_leauge','park_notes']
-    return park_df
+    park_url = "https://www.retrosheet.org/parkcode.txt"
+    try:
+        park_columns=['park_id','park_name','park_alt_name','park_city',
+        'park_state','park_start_date','park_end_date','park_league','park_notes']
+        park_df = pd.read_csv(park_url,sep=",",header=0,names=park_columns)
+        return park_df
+    except:
+        print('Could not downoad the MLB ballpark file from Retrosheet. Check your internet connection before running this function agian.')
```

spelling: `again`

---

Also: I would print the error generated here instead of swallowing it. Could be something other than internet connection issues potentially.